tests/tests_pytorch/loggers/test_csv.py

```diff
@@ -119,6 +120,20 @@ def test_fit_csv_logger(tmpdir):
     assert os.path.isfile(metrics_file)
 
 
+@RunIf(sklearn=True)
+def test_fit_csv_logger_remotefs():
+    dm = ClassifDataModule()
+    model = ClassificationModel()
+    logger = CSVLogger(save_dir="memory://test_fit_csv_logger_remotefs")
+    trainer = Trainer(
+        default_root_dir="memory://test_fit_csv_logger_remotefs", max_steps=10, logger=logger, log_every_n_steps=1
+    )
+    trainer.fit(model, datamodule=dm)
```

Comment:
WE don't need to `fit` an sklearn model to test the added functionality

Reply:
fixed